examples/albert/run_trainer.py

```diff
@@ -204,7 +204,6 @@ def print_lr(self, *args, **kwargs):
             return self.optimizer.scheduler.print_lr(*args, **kwargs)
 
     def step(self):
-        logger.debug("Called NoOpScheduler.step")
```

Comment:
This is one of the most frequent debug messages, however it is not very useful. We agreed to remove it.

Reply:
willdo